components/src/main/java/org/folio/rest/workflow/model/CompressFileTask.java

```diff
@@ -2,114 +2,72 @@
 
 import java.util.HashSet;
 import java.util.Set;
-
 import javax.persistence.Column;
 import javax.persistence.ElementCollection;
 import javax.persistence.Embedded;
 import javax.persistence.Entity;
 import javax.persistence.EnumType;
 import javax.persistence.Enumerated;
-
+import lombok.Getter;
+import lombok.Setter;
+import org.folio.rest.workflow.enums.CompressFileContainer;
+import org.folio.rest.workflow.enums.CompressFileFormat;
 import org.folio.rest.workflow.model.components.DelegateTask;
+import org.folio.rest.workflow.model.has.HasAsync;
+import org.folio.rest.workflow.model.has.HasInputOutput;
+import org.folio.rest.workflow.model.has.common.HasCompressFileTaskCommon;
 
 @Entity
```

Comment:
Can we use [@Data](https://projectlombok.org/features/Data) here and avoid all the other annotations?